src/components/shared/Validations.tsx

```diff
@@ -28,8 +33,30 @@ const numberValidationFormatters: Record<string, (value: unknown) => string> = {
   minLength: value => `>= ${value} characters`,
   maximum: value => `<= ${value}`,
   exclusiveMaximum: value => `< ${value}`,
-  maxItems: value => `< ${value} items`,
-  maxLength: value => `< ${value} characters`,
+  maxItems: value => `<= ${value} items`,
+  maxLength: value => `<= ${value} characters`,
+};
+
+const validationsFormatter = (name: string) => (value: unknown[] | unknown): ValidationFormat | null => {
+  const values = Array.isArray(value) ? value : [value];
+  if (values.length) {
+    return {
+      name: values.length > 1 ? `${name} values` : `${name} value`,
+      values: values.map(stringifyValue),
+    };
+  }
+  return null;
+};
+
+const validationFormatters: Record<string, (value: unknown) => ValidationFormat | null> = {
+  ['const']: validationsFormatter('Allowed'),
+  enum: validationsFormatter('Allowed'),
+  examples: validationsFormatter('Example'),
+  example: validationsFormatter('Example'),
+  ['x-example']: validationsFormatter('Example'),
+  multipleOf: validationsFormatter('Multiple of'),
+  pattern: validationsFormatter('Match pattern'),
```

Comment:
Your abstraction falls a bit short here. These two shouldn't say `values`.

I'm willing to let it slide for now though if @philsturgeon is, too.

cc @philsturgeon Is it acceptable that it currently says "Match pattern value" and "Multiple of value"?

Reply:
Oh yeah, right, it shouldn't. I'll think of better way